src/test/java/io/pravega/connectors/flink/utils/SetupUtils.java

```diff
@@ -274,25 +302,29 @@ public void start() throws Exception {
 
             this.inProcPravegaCluster = InProcPravegaCluster.builder()
                     .isInProcZK(true)
+                    .secureZK(enableTls)
                     .zkUrl("localhost:" + zkPort)
                     .zkPort(zkPort)
                     .isInMemStorage(true)
                     .isInProcController(true)
                     .controllerCount(1)
                     .restServerPort(restPort)
+                    .enableRestServer(true)
                     .isInProcSegmentStore(true)
                     .segmentStoreCount(1)
                     .containerCount(4)
+                    .enableAuth(enableAuth)
                     .enableTls(enableTls)
-                    .keyFile(getFileFromResource(KEY_FILE))
+                    .enableRestServer(enableRestServer)
```

Comment:
For readability, the calls can be grouped together, i.e, REST-related methods `restServerPort` and `enableRestServer` one after another. Ditto for auth and TLS related methods.

Reply:
I have grouped them logically in the order of ZK, controller, SS and security configurations for readability.